packages/airnode-deployer/terraform/airnode/aws/modules/function/main.tf

```diff
@@ -47,6 +50,7 @@ resource "aws_lambda_function" "lambda" {
   function_name                  = var.name
   handler                        = var.handler
   runtime                        = "nodejs14.x"
+  memory_size                    = 256
```

Comment:
Will we be able to configure this in the future?

Reply:
Yes, I think that's the plan. I'm pretty sure we already have a ticket for that but I can't find it (Jira being Jira...).